ci/cron/daily-compat.yml

```diff
@@ -310,3 +310,26 @@ jobs:
         else
           tell_slack "Daily tests failed ($(Build.SourceBranchName)): $COMMIT_LINK." "$(Slack.ci-failures-daml)"
         fi
+
+  - job: snapshots
+    timeoutInMinutes: 60
+    #condition: eq(variables['Build.SourceBranchName'], 'main')
```

Comment:
I assume this is deliberate?

Reply:
This is for testing this branch, but shouldn't be part of the merged end result.

Reply:
(as in, in the merged version, it should be uncommented)

Reply:
And of course I promptly forgot about this. Can you take a look at https://github.com/digital-asset/daml/pull/18925 when you have a minute please? Sorry for the noise.